manifests/lvs/virtual_server.pp

```diff
@@ -11,107 +11,115 @@
 # Refer to keepalived's documentation to understand the behaviour
 # of these parameters
 #
-# [*ip_address*]
+# @param ip_address
 #   Virtual server IP address.
 #
-# [*port*]
+# @param port
 #   Virtual sever IP port.
 #
-# [*fwmark*]
+# @param fwmark
 #   Virtual Server firewall mark. (overrides ip_address and port)
 #   Default: not set
 #
-# [*lb_algo*]
+# @param lb_algo
 #   Must be one of rr, wrr, lc, wlc, lblc, sh, dh
 #   Default: not set.
```

Comment:
Since we are using `puppet-string` the default values are auto-generated. Giving this information by a manual description can be not what is in the code. So please remove them.